stat/distuv/logistic.go

```diff
@@ -0,0 +1,98 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+)
+
+// Logistic implements the Logistic distribution, a two-parameter distribution with support on the real axis.
+// Its cumulative distribution function is the logistic function.
+//
+// General form of probability density fuction for Logistic distribution is
+//  E(x) / (s * (1 + E(x))^2)
+//  where E(x) = exp(-(x-μ)/s)
+//
+// For more information, see https://en.wikipedia.org/wiki/Logistic_distribution
```

Comment:
Final dot.

Reply:
Where do you want that? I deliberately skipped dot after link because dot is valid symbol in URL path and thus can be considered as part of URL so link will be invalid

Reply:
The pkg.go.dev site is smart enough to not include trailing dots (for example, see https://pkg.go.dev/cmd/go#hdr-Calling_between_Go_and_C and the link to swig.org there). Valid concern though.